o1vm/src/pickles/tests.rs

```diff
@@ -121,3 +125,31 @@ fn test_small_circuit() {
     );
     assert!(verif, "Verification fails");
 }
+
+#[test]
```

Comment:
Feel free to disagree with these changes. It is a single commit we can simply delete. I only want to keep a regression test.

Reply:
These are good to visualize the behavior when zeroes are involved.